metadata.yaml

```diff
@@ -37,6 +37,9 @@ resources:
   jenkins-image:
     type: oci-image
     description: OCI image for Jenkins
+    mounts:
```

Comment:
Isnt the storage definition missing?

Reply:
Thanks, storage definition is added. I'm surprised that tests still pass even without this

Reply:
@Thanhphan1147  I think the code needs to ship `storage_mounted` hook, I think what's happening could be that Jenkins gets loaded before storage gets mounted and hence the things appear to work fine.